axe_selenium_python/tests/test_axe.py

```diff
@@ -4,45 +4,51 @@
 
 from os import path
 import pytest
+from axe_selenium_python import Axe
 
 
 @pytest.mark.nondestructive
-def test_get_rules(axe):
+def test_get_rules(selenium):
     """Assert number of rule tests matches number of available rules."""
+    axe = Axe(selenium)
```

Comment:
This could probably get it's own fixture since it seems to be called for each test.